Extremem/src/main/java/com/amazon/corretto/benchmark/extremem/RelativeTime.java

```diff
@@ -259,6 +259,15 @@ RelativeTime divideBy(ExtrememThread t, int divisor) {
     return new RelativeTime(t, s, (int) ns);
   }
 
+  /**
+   * Return a primitive double representing the quotient of dividing this by divisor.
+   */
+  double divideBy(ExtrememThread t, RelativeTime divisor) {
```

Comment:
Does this need to take `ExtrememThread t`?

Reply:
You're right.  I'll remove that.  Returning a primitive so no "need" for an "allocation context".

Reply:
No need for thread argument because result is primitive double.  Thanks for that catch.